app/src/main/java/org/inspire/breath/data/MainDao.java

```diff
@@ -9,18 +9,16 @@
 @Dao
 public interface MainDao {
 
-
     @Insert
-    void insertVolunteer(Volunteer volunteer);
+    void insertVolunteer(Patient volunteer);
 
-    @Query("DELETE FROM volunteer_table")
+    @Query("DELETE FROM patients")
     void deleteAllVolunteers();
     
-    @Query("SELECT * FROM volunteer_table ORDER BY lastName ASC")
-    List<Volunteer> getAllVolunteers();
-
-    @Query("SELECT * FROM volunteer_table WHERE id = :id")
-    List<Volunteer> getVolunteerById(int id);
+    @Query("SELECT * FROM patients ORDER BY lastName ASC")
+    List<Patient> getAllVolunteers();
```

Comment:
Some of the methods here still refer to volunteer, Is it possible we can rename these too please :)